packages/block-editor/src/components/block-inspector/README.md

```diff
@@ -16,6 +16,16 @@ import { BlockInspector } from '@wordpress/block-editor';
 const MyBlockInspector = () => <BlockInspector />;
 ```
 
+### Props
+
+#### showNoBlockSelectedMessage
+
+Whether to display a "No block selected" message when no block is selected.
+
+- Type: `boolean`
+- Required: No
+- Default: `true`
```

Comment:
```suggestion
-   Type: `boolean`
-   Required: No
-   Default: `true`
```

Nit: Note that 3 spaces are recommended in the list instead of 1.

However, CI currently does not perform auto-formatting on Markdown files. See #66235 for details.